### PR TITLE
Hyper-V section name has changed

### PR DESCRIPTION
**Pull request by mcgov**
Hyper-V CORE AND DRIVERS -> Hyper-V/Azure CORE AND DRIVERS

### CODE REVIEW

File: Util/Config.py
Comment (andschwa) [inline Util/Config.py:9]:
```suggestion
sections = ["Hyper-V CORE AND DRIVERS", "Hyper-V/Azure CORE AND DRIVERS"]
```

I would suggest having both the old and new names to cover the case where downstream distros don't immediately update the maintainers file. But I haven't checked and that may not actually be a concern.

**Comment by mcgov** [comment]
Going to add another section for the hyper-v DRM video driver, hold this for one last commit plz

**Comment by mcgov** [comment]
Commit is added! Ready for review and merge

**Review by andschwa — APPROVED** [review]

**Comment by mcgov** [comment]
Just a heads up, I can't merge this one. It seems like the pipeline is broken, can we remove it and add me to the project? I'll get a new pipeline set up or something.

**Comment by andschwa** [comment]
And while I still had permissions to add @mcgov to the "Linux Systems Group" GitHub team, I don't have admin permissions on the repo to either re-connect the CI pipeline or remove it as a requirement.

**Pull request merged**
